### ISSUE TITLE
There are 2 same cpoied blob containers unless refresh in the unexpaned blob containers with no blobs in the Azure and Azure Stack

### ISSUE THREAD

**Issue by xin42** [Microsoft/AzureStorageExplorer#1254]
**Storage Explorer Version**: 1.7.0
**Platform/OS**: Windows 10

**Bug description**
there are two copied blob container shows in the blob containers unless refresh

Steps to Reproduce
1. Install corresponding SE;
2. Create two storage accounts(A,B) in the admin portal and two storage accounts(C,D) in the tenant portal; 
3. Sign in with server admin account and tenant account;
4. Click the storage account A in the Storage Explorer, create a new blob container  with name 'test';
5. Open Storage account B in the SE, then copy 'test' into Storage account B's Blob containers(Be cautious, please don't click the Blob containers expanded button before this);
6. Click Load more;
7. You can repeat the steps 4-6 for tenant storage account; 

**Expected Experience**
After step 6, there is a only one copied blob container in the Storage account B.

**Actual Experience**
After step 6, there is two same copied blob containers in the Storage account B unless refresh.

**Comment by MRayermannMSFT** [comment]
Closing as won't fix. Issue isn't related to Azure Stack directly. This is a edge case in the tree view "prevent duplication" code. The container that is created on paste is given a different UID then the one we end up getting from the server. Thus there are duplicates. The fix would be to make sure the container that paste creates its UID in the same way that a container from the server creates its.
